todo_CLI_3/todo_CLI.js

```diff
@@ -0,0 +1,82 @@
+
+const readline = require('readline');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  prompt:">"
+})
+
+let listArray= [];
+let a = false;
```

Comment:
I'm seeing you are using `a` as a flag to indicate it's a create request. I think I've mentioned this in the previous pr but please be descriptive with the variable name - kinda hard to tell the purpose at first glance.
Also, not sure if `a` is required. You can include the logic in 25-27 in your `newFunc`.